tests/constants/admincli/cmd.go

```diff
@@ -25,4 +25,9 @@ const (
 
 	// ListVolumes referring to vmdkops_admin volume ls
 	ListVolumes = vmdkopsAdminVolume + "ls "
+
+	//DetachedStatus - volume detached
```

Comment:
admincli is not better home .. please move this const out .. it leads to confusion.